msgboard/templates/msgboard/board.html

```diff
@@ -18,11 +13,12 @@ <h1>Beyond message board</h1>
 </div>
 {% for message in messages %}
     <div class="message">
-   	 <p class="author">
-   		 Posted by {{message.author}} at {{message.date}}
-   	 </p>
-   	 <p>{{ message.text }}</p>
+         <p class="author">
```

Comment:
not seeing any code changes, are you changing the indention style? please make sure your editor does not use tab characters.

Reply:
Ack